docs/templates/base.html

```diff
@@ -0,0 +1,144 @@
+{% extends "adidoks/templates/base.html" %}
+{% block footer %}
+{{ super() }}
+<script>
+document.addEventListener("DOMContentLoaded", inject);
+
+function inject() {
+  document.querySelectorAll(".code-selector").forEach(injectTabs);
+}
+
+function injectTabs(codeSelector) {
+  const nav = codeSelector.querySelector("ul.nav");
+
+  let langs = [];
+  let tabs = [];
```

Comment:
These two can be `const`s. It's fine for the value to mutate, as long as it doesn't get pointed to a new object

Reply:
My new editor doesn't have an inbuilt linter 😭 

I'll fix all these up.